cosi.proto

```diff
@@ -97,22 +150,13 @@ message ProvisionerCreateBucketRequest {
 
     // This field is OPTIONAL
     // Protocol specific information required by the call is passed in as key,value pairs.
```

Comment:
Add a comment indicating 1) caller should treat these as opaque, 2) receiver is responsible for parsing and validating.